services/api-server/src/simcore_service_api_server/modules/director_v2.py

```diff
@@ -94,7 +94,7 @@ async def create_computation(
         self, project_id: UUID, user_id: PositiveInt
     ) -> ComputationTaskOut:
         resp = await self.client.post(
-            "/computations",
+            "/v2/computations",
```

Comment:
maybe put this `v2` as class constant on `DirectorV2Api` so you can use it like `DirectorV2Api.API_VERSION`

Reply:
this API has BOTH v1 and v2 entries (until director is fully deprecated)